### ISSUE TITLE
Looking for missing branch codedust from Signal-Desktop for web browser

### ISSUE THREAD

**Issue by serifreedom** [codedust/Textsecure-Browser#2]
Hi codedust ,
Is this repo the same as https://github.com/WhisperSystems/Signal-Desktop/pulls/codedust
from https://github.com/WhisperSystems/Signal-Desktop/issues/723

sorry i don't know how to contact you, then i have to ask  by  new this issue.

**Comment by codedust** [comment]
Correct. You're in the right repo. However, these PRs are quite old so I don't think it's worth it rebasing them on the current master.
